core_lib/src/structure/objectdata.h

```diff
@@ -65,7 +67,7 @@ class ObjectData
     QTransform mCurrentView;
     
     // playback manager
-    int  mFps    = 12;
+    int  mFps;
```

Comment:
same comment, this should have a value to avoid undefined behaviours in case the fps won't be set.